app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    result = command.split(" ")
+    if result[1] == result[2]:
+        return
+    with open(result[1], "r") as file_in, open(result[2], "w") as file_out:
+        file_out.write(file_in.read())
```

Comment:
no need to make blank return make a backwar condition
u can use unpacking for split func like this: "cp, file1, file2 = command.split() <- no need to add space to split func it split by space by default
also u should check if cp == "cp"

Reply:
Done